user/example-apps/koracle/models/oracle/oracle.py

```diff
@@ -2,13 +2,10 @@
 
 import pandas as pd
 
-import __init__
-from h1st.model.rule_based_model import RuleBasedModel
+from h1st.model.knowledge_model import RuleBasedModel
```

Comment:
Shouldn’t this be removed entirely? It belongs in `k1st`

Reply:
I removed this one.
This has been moved to k1st